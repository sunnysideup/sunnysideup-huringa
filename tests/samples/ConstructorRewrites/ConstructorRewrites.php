<?php

class ConstructorRewrites {
function ConstructorRewrites() {}
function MyOtherFunction() {}
}