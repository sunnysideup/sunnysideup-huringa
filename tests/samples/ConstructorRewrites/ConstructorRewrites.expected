<?php

class ConstructorRewrites {
function __construct() {}
function MyOtherFunction() {}
}
